engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/metrics/InstantRateMeter.scala

```diff
@@ -1,26 +1,27 @@
 package pl.touk.nussknacker.engine.flink.util.metrics
 
 import cats.data.NonEmptyList
-import org.apache.flink.metrics.{Counter, Gauge}
-import pl.touk.nussknacker.engine.util.metrics.RateMeter
+import org.apache.flink.metrics.Gauge
+import pl.touk.nussknacker.engine.util.metrics.{MetricIdentifier, MetricsProvider, RateMeter}
 
 //TODO: add also "normal" rate meter here (such as in dropwizard etc)
 class InstantRateMeter extends pl.touk.nussknacker.engine.util.metrics.GenericInstantRateMeter with Gauge[Double]
 
 object InstantRateMeterWithCount {
 
-  def register(tags: Map[String, String], name: List[String], metricUtils: MetricUtils) : InstantRateMeterWithCount = {
-    val rateMeter = metricUtils.gauge[Double, InstantRateMeter](NonEmptyList.ofInitLast(name, "instantRate"), tags, new InstantRateMeter)
-    val counter = metricUtils.counter(NonEmptyList.ofInitLast(name, "count"), tags)
-    new InstantRateMeterWithCount(rateMeter, counter)
+  def register(tags: Map[String, String], name: List[String], metricUtils: MetricsProvider) : InstantRateMeterWithCount = {
```

Comment:
can you replace `metricUtils: MetricProvider` occurrences with `metricProvider: MetricProvider` in the procject?

Reply:
done